daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -132,15 +132,19 @@ cmdBuild numProcessors =
     info (helper <*> cmd) $
     progDesc "Initialize, build and package the DAML project" <> fullDesc
   where
-    cmd = execBuild numProcessors <$> optionalOutputFileOpt
+    cmd =
+        execBuild <$> optionsParser numProcessors (pure Nothing) <*>
+        optionalOutputFileOpt
```

Comment:
That is a slightly misleading way to break lines. Can we please do
```
func
  <$> arg1
  <*> arg2
```
Same below.

Reply:
it's autoformatted by hindent.